search/src/main/scala/weco/api/search/services/ImagesRequestBuilder.scala

```diff
@@ -36,29 +36,34 @@ class ImagesRequestBuilder(queryConfig: QueryConfig)
     binMinima = queryConfig.paletteBinMinima
   )
 
-  def request(searchOptions: ImageSearchOptions, index: Index): SearchRequest =
-    search(index)
-      .aggs { filteredAggregationBuilder(searchOptions).filteredAggregations }
-      .query(searchQuery(searchOptions))
-      .sortBy { sortBy(searchOptions) }
-      .limit(searchOptions.pageSize)
-      .from(PaginationQuery.safeGetFrom(searchOptions))
-      .sourceInclude(
-        "display",
-        // we do KNN searches for similar images, and for that we need
-        // to send the image's vectors to Elasticsearch
-        "query.inferredData.reducedFeatures"
-      )
-      .postFilter {
-        must(buildImageFilterQuery(searchOptions.filters))
-      }
+  def request(
+    searchOptions: ImageSearchOptions,
+    index: Index
+  ): Left[SearchRequest, Nothing] =
+    Left(
```

Comment:
You may wish to hide whitespace when reviewing.  The only difference here is that the SearchRequest is now returned in  a Left